pkg/otlp/internal/serializerexporter/exporter.go

```diff
@@ -40,16 +56,52 @@ type exporter struct {
 	s  serializer.MetricSerializer
 }
 
-func newExporter(logger *zap.Logger, s serializer.MetricSerializer) (*exporter, error) {
-	// TODO (AP-1267): Expose these settings in datadog.yaml.
-	tr, err := translator.New(logger,
+func translatorFromConfig(logger *zap.Logger, cfg *exporterConfig) (*translator.Translator, error) {
+	histogramMode := translator.HistogramMode(cfg.Metrics.HistConfig.Mode)
+	switch histogramMode {
+	case translator.HistogramModeCounters, translator.HistogramModeNoBuckets, translator.HistogramModeDistributions:
+		// Do nothing
+	default:
+		return nil, fmt.Errorf("invalid `mode` %s", cfg.Metrics.HistConfig.Mode)
+	}
+
+	options := []translator.Option{
 		translator.WithFallbackHostnameProvider(hostnameProviderFunc(util.GetHostname)),
-		translator.WithHistogramMode(translator.HistogramModeDistributions),
-		translator.WithNumberMode(translator.NumberModeCumulativeToDelta),
-		translator.WithQuantiles(),
-	)
+		translator.WithHistogramMode(histogramMode),
+		translator.WithDeltaTTL(cfg.Metrics.DeltaTTL),
+	}
+
+	if cfg.Metrics.HistConfig.SendCountSum {
+		options = append(options, translator.WithCountSumMetrics())
+	}
+
+	if cfg.Metrics.Quantiles {
+		options = append(options, translator.WithQuantiles())
+	}
+
+	if cfg.Metrics.ExporterConfig.ResourceAttributesAsTags {
+		options = append(options, translator.WithResourceAttributesAsTags())
+	}
+
+	if cfg.Metrics.ExporterConfig.InstrumentationLibraryMetadataAsTags {
+		options = append(options, translator.WithInstrumentationLibraryMetadataAsTags())
+	}
+
+	var numberMode translator.NumberMode
+	if cfg.Metrics.SendMonotonic {
+		numberMode = translator.NumberModeCumulativeToDelta
+	} else {
+		numberMode = translator.NumberModeRawValue
+	}
+	options = append(options, translator.WithNumberMode(numberMode))
+
+	return translator.New(logger, options...)
+}
+
+func newExporter(logger *zap.Logger, s serializer.MetricSerializer, cfg *exporterConfig) (*exporter, error) {
+	tr, err := translatorFromConfig(logger, cfg)
 	if err != nil {
-		return nil, fmt.Errorf("failed to build translator: %w", err)
+		return nil, err
```

Comment:
Maybe something like this?
```suggestion
		return nil, fmt.Errorf("incorrect OTLP metrics configuration: %w", err)
```